src/language/constructors.md

```diff
@@ -96,11 +97,6 @@ class Point {
   final double y;
 
   Point(this.x, this.y);
-
```

Comment:
This section is about named constructors but now there isn't a named constructor :). Fwiw there wasn't anything wrong with this constructor (this is the "initializer list" style I was talking about above)

Reply:
Oops, that was just an accident! Lots of layered excerpts for different examples in that code 😅